uefi/gberegion.go

```diff
@@ -18,6 +22,19 @@ func NewGBERegion(buf []byte, r *Region) (*GBERegion, error) {
 	return &gbe, nil
 }
 
+// Validate Region
+func (gbe *GBERegion) Validate() []error {
+	// TODO: Add more verification if needed.
+	errs := make([]error, 0)
+	if gbe.Position == nil {
+		errs = append(errs, fmt.Errorf("GBERegion position is nil"))
+	}
+	if !gbe.Position.Valid() {
+		errs = append(errs, fmt.Errorf("GBERegion is not valid, region was %v", gbe.Position))
```

Comment:
if I'm not wrong you will have the pointer of `gbe.Position` in this format string. Is this what you want?

(Same question for the other regions below)

Reply:
Nice catch! Thanks!